scripts/generateAssetData/overrides.ts

```diff
@@ -3,7 +3,7 @@ import type { Asset } from '@shapeshiftoss/types'
 
 const fox: Partial<Asset> = {
   color: '#3761F9',
-  icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+  icon: 'https://assets.coingecko.com/coins/images/9988/standard/FOX.png?1696510025',
```

Comment:
```suggestion
  icon: 'https://coin-images.coingecko.com/coins/images/9988/large/FOX.png?1696510025',
```

Reply:
Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)